Source/Core/Core/HW/Wiimote.cpp

```diff
@@ -135,21 +135,37 @@ void Pause()
 // An L2CAP packet is passed from the Core to the Wiimote on the HID CONTROL channel.
 void ControlChannel(int number, u16 channel_id, const void* data, u32 size)
```

Comment:
Wouldn't it be kinda nice if we could _just_ call `s_config.GetController(number)->ControlChannel(...)` and not worry about whether it is a real, emulated or...disabled/non-existant Wiimote? Goes for both `ControlChannel` and `InterruptChannel`.